draft-ietf-jsonpath-base.md

```diff
@@ -1153,17 +1153,24 @@ resulting in an empty nodelist, or
 When any path on either side of a comparison results in a nodelist consisting of a single node, each such path is
 replaced by the value of its node and then:
 
-* comparison using one of the operators `==` yields true if and only if the comparison
-is between equal primitive values.
+* a comparison using the operator `==` yields true if and only if the comparison
```

Comment:
I'd prefer to avoid redundancy in the normative text. How would you feel about some notes pointing out these implications?

Reply:
I'm troubled, because this very issue was at the center of our discussion, with dozens and dozen of messages, and the core *principle* underlying the spec is that comparisons between different types returns false.  So I think it's helpful to understanding why things are the way they are.  I think that if that is not explicitly stated, it weakens the clarity and usefulness of the text.

Reply:
Good point. I'll try reworking to include those type statements.

Reply:
I agree with @timbray here that it's better to have an explicit statement.  Implicitness is often seen as subjective, and that's not what we want.

Reply:
Fixed in e2f005d55adeadb764696b24999b5d40172c8b2c.